### PR TITLE
Improve the layout of edit coach form

### PR DESCRIPTION
**Pull request by rodrigo-morais**
Creating a message to be presented when a coach is save in Edit form. It can show an error or success message.

### CODE REVIEW

**Comment by rodrigo-morais** [comment]
ping @viniciusffj

File: src/Coaches/Edit/Edit.elm [hunk 1]
```diff
@@ -29,21 +59,21 @@ nav address model =
...
       else
-        model.name
+        model.coach.name
 
   in
     div [ class "m3" ]
         [
           h1  []
               [ text name ],
-          formSpots address model,
```
Comment (viniciusffj) [inline src/Coaches/Edit/Edit.elm]: I guess you forgot to update this :)

File: src/Coaches/Edit/Edit.elm [hunk 2]
```diff
@@ -29,21 +59,21 @@ nav address model =
...
+          formSpot address model,
```
Comment (viniciusffj) [inline src/Coaches/Edit/Edit.elm:76]: I guess you forgot to update this

**Comment by viniciusffj** [comment]
@rodrigo-morais Looks good

I just updated the commit message because it was igual the one we have in #5

**Pull request closed without merging**